packages/application/src/submission.tsx

```diff
@@ -157,3 +166,62 @@ export class SubmissionHandler {
     );
   }
 }
+
+interface IProps {
+  reason: string;
+  message: string;
+  traceback: string;
+  default_msg: string;
+}
+
+class ErrorDialogContent extends React.Component<IProps, any> {
+  constructor(props: any) {
+    super(props);
+    this.state = { expanded: false };
+  }
+
+  toggleMsgDisplay(): void {
+    // Switch expanded flag
+    const expanded = !this.state.expanded;
+    this.setState({ expanded: expanded });
+  }
+
+  render(): React.ReactElement {
+    const details = this.props.traceback ? (
+      <div>
+        <div>
+          <button
+            className={
+              ERROR_DISPLAY_BUTTON +
+              ' ' +
+              (this.state.expanded ? UP_ICON_CLASS : DOWN_ICON_CLASS)
+            }
+            onClick={(): void => {
+              this.toggleMsgDisplay();
+            }}
+          ></button>
+          {'Error details: '}
+        </div>
+        <br />
+        <div
+          className={
+            this.state.expanded ? ERROR_DETAILS_VISIBLE : ERROR_DETAILS_HIDDEN
+          }
+        >
+          {this.props.traceback}
+        </div>
+      </div>
+    ) : null;
+
+    return (
+      <div className={MESSAGE_DISPLAY}>
+        {this.props.reason + ': ' + this.props.message}
+        <br />
+        <br />
```

Comment:
does including all these breaks make sense if `details == null`? If not maybe they should be part of `details`

Reply:
Good catch. The first adds a new line to message, the second adds a blank line. 
We could move one of them to `details`, so if we get an empty traceback it looks like 
![image](https://user-images.githubusercontent.com/25207344/76904052-9811ff00-6875-11ea-8d94-e781c94c08b3.png)